<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="IDC_STATIC4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>296</height>
     </rect>
    </property>
    <property name="title">
     <string>Fingerprint Operation:</string>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="IDC_STATIC13">
         <property name="text">
          <string>Sensor Count:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IDC_EDCount">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="IDC_STATIC1">
         <property name="text">
          <string>Current:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IDC_EDUsed">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="IDC_STATIC2">
         <property name="text">
          <string>Serial Number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IDC_EDASN">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="IDC_RADIO9">
           <property name="text">
            <string>ZKFinger 9.0</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="IDC_RADIO10">
           <property name="text">
            <string>ZKFinger 10.0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="IDC_BTNInit">
           <property name="text">
            <string>Connect Sensor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="IDC_BTNDisConnect">
           <property name="text">
            <string>Disconnect Sensor</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="IDC_BTNReg">
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="IDC_BTNSave">
         <property name="text">
          <string>Save Image</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QGroupBox" name="IDC_STATIC3">
       <property name="title">
        <string>Image Format:</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QRadioButton" name="IDC_RADIOBMP">
            <property name="text">
             <string>BMP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="IDC_RADIOJPG">
            <property name="text">
             <string>JPG</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="IDC_BTNVer">
           <property name="text">
            <string>Verify (1:1)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="IDC_BTNIdentify">
           <property name="text">
            <string>Identify(1:N)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="IDC_BTNREGBYIMAGE">
           <property name="text">
            <string>Register by image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="IDC_BTNIDENBYIMAGE">
           <property name="text">
            <string>Identify by image</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="5" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="IDC_BTNRED">
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="IDC_BTNGREEN">
         <property name="text">
          <string>Green</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="IDC_BTNBEEP">
         <property name="text">
          <string>Beep</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="test">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="IDC_STATIC5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>RFID Operation:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="IDC_STATIC10">
         <property name="text">
          <string>Pin:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IDC_EDIT_FINGERPIN"/>
       </item>
       <item>
        <widget class="QCheckBox" name="IDC_PINWIDE5">
         <property name="text">
          <string>PIN_WIDTH=5</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="IDC_BUTTON5">
           <property name="text">
            <string>WriteTemplateToCard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="IDC_BUTTON6">
           <property name="text">
            <string>ReadTemplateFromCard</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPushButton" name="IDC_BUTTON1">
           <property name="text">
            <string>ReaderSerialNumber</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="IDC_BUTTON2">
           <property name="text">
            <string>CardNumber</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="IDC_STATIC6">
            <property name="text">
             <string>Block:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="IDC_EDIT_Block"/>
          </item>
          <item>
           <widget class="QPushButton" name="IDC_BUTTON3">
            <property name="text">
             <string>ReadCard</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="IDC_BTNREADPWD">
            <property name="text">
             <string>ReadPass</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="IDC_STATIC7">
            <property name="text">
             <string>Block:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="IDC_EDIT_WriteBlock"/>
          </item>
          <item>
           <widget class="QPushButton" name="IDC_BUTTON4">
            <property name="text">
             <string>WriteCard</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="IDC_BTNWRITEPWD">
            <property name="text">
             <string>WritePass</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="IDC_STATIC11">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Init-&gt;Register-&gt;WriteTemplateToCard</string>
    </property>
   </widget>
   <widget class="QLabel" name="IDC_STATIC12">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>510</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Init-&gt;ReadTemplateFromCard-&gt;Identify</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="IDC_EDHint">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>550</y>
      <width>701</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>341</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
